test/lib/instructions-parser.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { getPackagesFromRunInstructions } from "../../lib/dockerfile/instruction-parser";
+
+describe("instruction parser", () => {
+  const cases = [
+    [["apt install curl"], ["curl"]],
+    [["apt-get install curl"], ["curl"]],
+    [["apt-get -y install curl"], ["curl"]],
+    [["aptitude install curl"], ["curl"]],
+    [["yum install curl"], ["curl"]],
+    [["apk add curl"], ["curl"]],
+    [["apk --update add curl"], ["curl"]],
+    [["rpm -i curl"], ["curl"]],
+    [["rpm --install curl"], ["curl"]],
+    [["apt-get install -y wget curl -V"], ["curl", "wget"]],
+    [["    apt   install   curl   vim   "], ["curl", "vim"]],
+    [["apt install curl  vim"], ["curl", "vim"]],
+    [["apt install curl wget vim"], ["vim", "curl", "wget"]],
+    [["apt install curl && apt install wget"], ["curl", "wget"]],
+    [['apt install curl; apt install vim; echo "bitwise"'], ["curl", "vim"]],
+    [
+      ["apt install curl && apt       install vim", "apt install   -y  wget"],
+      ["curl", "vim", "wget"],
+    ],
+    [["apt install 389-admin"], ["389-admin"]],
```

Comment:
Just checking that things like nvm install, rvm install etc are out of scope for us?

Reply:
Also, what about curl commands that are piped into bash, eg `RUN \curl -sSL https://get.rvm.io | bash -s stable`?

Reply:
we support only commands that use the package manager :) so only `apt`/`rpm`/`apk` commands for now. regarding `curl`ing - we partially use nodejs/java that is installed that way with the key binaries feature